services/apps/alcs/src/queues/scheduler/scheduler.service.spec.ts

```diff
@@ -92,7 +103,7 @@ describe('SchedulerService', () => {
   //     {
   //       repeat: {
   //         pattern: isDST()
-  //           ? MONDAY_TO_FRIDAY_AT_2AM_PDT_IN_UTC 
+  //           ? MONDAY_TO_FRIDAY_AT_2AM_PDT_IN_UTC
   //           : MONDAY_TO_FRIDAY_AT_2AM_PST_IN_UTC,
   //       },
   //     },
```

Comment:
Maybe we should remove these comments?

Reply:
That's for a job we built but disabled as per Ansleys request. This test covers the disabled job.
  //Job Disabled for now